content/general/limits.textile

```diff
@@ -0,0 +1,314 @@
+
+If the channel creation rate is exceeded, a "global per second limit":#global-time will be applied to channel creation.
+
+h3(#message-publish-rate). Message publish rate
+
+The message rate is the maximum rate at which messages can be published per second for each channel.
+
+If the message publish rate is exceeded for a channel, a "local per second limit":#time on publishing will be applied to that channel.
+
+h2(#message). Message limits
+
+Message limits relate to the number, rate and bandwidth of "messages":/realtime/messages on your account.
+
+|_=. Limit |_=. Free |_=. Self-service |_=. Business |_=. Enterprise |
+| "Message size":#message-size (KiB) | 16 | 64 | 64 | 256 |
+| "History TTL":#history-ttl (hours) | 24 | 72 | 72 | Custom |
+
+h3(#message-size). Message size
+
+The message size is the maximum size of a single published "message":/glossary#messages.
+
+Attempting to publish a message larger than the message size limit will fail and the error code @40009@ will be returned.
+
+h3(#history-ttl). History TTL
+
+History time to live (TTL) is the maximum time that a message or presence event can be retrieved from "history":/realtime/history.
+
+h2(#queues). Queue limits
+
+Queue limits relate to the number, length and rates of "queues":/glossary#queues.
+
+|_=. Limit |_=. Free |_=. Self-service |_=. Business |_=. Enterprise |
+| "Number of queues":#queue-numbers (per account) | 5 | 50 | 50 | Custom |
+| "Queue length":#queue-length (per account) | 10,000 | 50,000 | 50,000 | Custom |
+| "Queue publish rate":queue-publish (per second) | 100 | 200 | 200 | Custom |
+| "Queue TTL":#queue-ttl (hours) | 1 | 24 | 24 | Custom |
+
+h3(#queue-numbers). Number of queues
+
+The number of queues is the maximum number of queues that can be created. This limit includes all applications in your account.
+
+h3(#queue-length). Queue length
+
+Queue length is the maximum number of messages that can be stored in queues whilst waiting to be consumed. This value is shared between all queues on the account.
+
+For example, with a free account you could have one queue with a length of 10,000 messages, or two queues each with a length of 5,000 messages.
+
+h3(#queue-publish). Queue publish rate
+
+The queue publish rate is the maximum rate at which messages can be published to a queue, per second.
+
+If the queue publish rate is exceeded, a "global per second limit":#global-time on publishing to queues is applied.
+
+h3(#queue-ttl). Queue TTL
+
+Queue time to live (TTL) is the time that a message is stored in a queue for. If the message is not consumed before this time then it is transferred to the "dead letter queue":/glossary#dead-letter.
+
+h2(#integration). Integration limits
+
+Integration limits relate to the rate of "events":/general/events and the rate of messages streamed through "Reactor Firehose":/general/firehose.
+
+|_=. Limit |_=. Free |_=. Self-service |_=. Business |_=. Enterprise |
```

Comment:
This is correct for now but ongoing discussions about scaling these limits with messages. Worth keeping an eye out for changes here.

Reply:
Thanks for the heads up - this will be easy to change once published.